tools/archetype/binding/src/main/resources/archetype-resources/ESH-INF/binding/binding.xml

```diff
@@ -1,8 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<binding:binding id="${bindingId}"
-				 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-				 xmlns:binding="http://eclipse.org/smarthome/schemas/binding/v1.0.0"
-				 xsi:schemaLocation="http://eclipse.org/smarthome/schemas/binding/v1.0.0 http://eclipse.org/smarthome/schemas/binding-1.0.0.xsd">
+<binding:binding id="mytestbinding" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"

```

Comment:
The `id="${bindingId}"` got substituted :-)

Reply:
Ups, sloppy mistake. Fixed it. Thanks for the hint.